frontend/src/lib/components/FlagSelector.tsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from 'react'
+import { useValues } from 'kea'
+import { featureFlagLogic } from 'scenes/feature-flags/featureFlagLogic'
+import { TaxonomicFilterGroupType, TaxonomicFilterLogicProps } from 'lib/components/TaxonomicFilter/types'
+import { Popover } from 'lib/lemon-ui/Popover'
+import { TaxonomicFilter } from 'lib/components/TaxonomicFilter/TaxonomicFilter'
+import { LemonButton } from 'lib/lemon-ui/LemonButton'
+
+interface FlagSelectorProps {
+    value: number | undefined
+    onChange: (id: number, key: string) => void
+    readOnly?: boolean
+}
+
+export function FlagSelector({ value, onChange, readOnly }: FlagSelectorProps): JSX.Element {
+    const [visible, setVisible] = useState(false)
+
+    const { featureFlag } = useValues(featureFlagLogic({ id: value || 'link' }))
+
+    const taxonomicFilterLogicProps: TaxonomicFilterLogicProps = {
+        groupType: TaxonomicFilterGroupType.FeatureFlags,
+        value: value,
+        onChange: (_, __, item) => {
+            'id' in item && item.id && onChange(item.id, item.key)
```

Comment:
@liyiy @neilkakkar I've moved the flag selector into components since I wanted to use it in replay config. And exposed the flag key as well as the id on change... That ok?